File: app/src/test/java/com/mgaetan89/showsrage/adapter/EpisodePagerAdapter_EmptyTest.java
package com.mgaetan89.showsrage.adapter;

import android.support.v4.app.Fragment;

import org.junit.After;
import org.junit.Before;
import org.junit.Test;

import java.util.Collections;

import static org.assertj.core.api.Assertions.assertThat;
import static org.mockito.Mockito.mock;

public class EpisodePagerAdapter_EmptyTest {
	private EpisodePagerAdapter adapter;

	@Before
	public void before() {
		this.adapter = new EpisodePagerAdapter(null, mock(Fragment.class), Collections.<Integer>emptyList());
	}

	@Test
	public void getCount() {
		assertThat(this.adapter.getCount()).isEqualTo(0);
	}

	@After
	public void after() {
		this.adapter = null;
	}
}

File: app/src/test/kotlin/com/mgaetan89/showsrage/adapter/EpisodePagerAdapter_EmptyTest.kt
package com.mgaetan89.showsrage.adapter

import android.support.v4.app.Fragment
import org.assertj.core.api.Assertions.assertThat
import org.junit.Before
import org.junit.Test
import org.mockito.Mockito.mock

class EpisodePagerAdapter_EmptyTest {
	private lateinit var adapter: EpisodePagerAdapter

	@Before
	fun before() {
		this.adapter = EpisodePagerAdapter(null, mock(Fragment::class.java), emptyList<Int>())
	}

	@Test
	fun getCount() {
		assertThat(this.adapter.count).isEqualTo(0)
	}
}
